aspnetcore/host-and-deploy/blazor/server-side.md

```diff
@@ -22,19 +22,34 @@ With the [server-side hosting model](xref:blazor/hosting-models#server-side), Bl
+Plan a deployment to make the best use of the available infrastructure for a Blazor Server app. See the following resources to address Blazor Server app scalability:
 
-## SignalR configuration
+* [Fundamentals of Blazor Server apps](xref:blazor/hosting-models#server-side)
+* <xref:security/blazor/server-side>
 
-SignalR is configured by ASP.NET Core for the most common Blazor server-side scenarios. For custom and advanced scenarios, consult the SignalR articles in the [Additional resources](#additional-resources) section.
+### Deployment server
 
-## Additional resources
+When considering the scalability of a single server (scale up), the memory available to an app is likely the first resource that the app will exhaust as user demands increase. The available memory on the server affects the:
 
-* <xref:signalr/introduction>
-* [Azure SignalR Service Documentation](/azure/azure-signalr/)
-* [Quickstart: Create a chat room by using SignalR Service](/azure/azure-signalr/signalr-quickstart-dotnet-core)
-* <xref:host-and-deploy/index>
-* <xref:tutorials/publish-to-azure-webapp-using-vs>
-* [Deploy ASP.NET Core preview release to Azure App Service](xref:host-and-deploy/azure-apps/index#deploy-aspnet-core-preview-release-to-azure-app-service)
+* Number of active circuits that a server can support.
+* UI latency on the client.
+
+For guidance on building scalable Blazor server apps, see <xref:security/blazor/server-side>.
+
+Each circuit uses approximately 250 KB of memory for a minimal *Hello World*-style app. The size of a circuit depends on the app's code and the state maintenance requirements associated with each component. We recommend that you measure resource demands during development for your app and infrastructure, but the following baseline can be a starting point in planning your deployment target: If you expect your app to support 5,000 concurrent users, consider budgeting at least 1.3 GB of server memory to the app (or ~273 KB per user).
+
+### SignalR configuration
+
+Blazor Server apps use ASP.NET Core SignalR to communicate with the browser. [SignalR's hosting and scaling conditions](xref:signalr/publish-to-azure-web-app) apply to Blazor Server apps.
+
+Blazor works best when using WebSockets as the SignalR transport due to lower latency, reliability, and [security](xref:signalr/security). Long polling is used by SignalR when WebSockets isn't available or when configured otherwise. When deploying to Azure App Service, configure the app to use WebSockets in the Azure portal settings for the service. For details on configuring the app for Azure App Service, see the [SignalR publishing guidelines](xref:signalr/publish-to-azure-web-app).
+
+<!-- 
+
+The phrase "or when configured otherwise" requires more specific explanation. It probably doesn't mean "when using a different transport than WebSockets." If it meant that, then the whole sentence wouldn't make sense (i.e., long polling would always be used).
```

Comment:
I wanted to say that you could force Blazor to use `LongPolling` which is what Azure App Services defaults to or you could force it by configuring the SignalR client in your application. 

> or when the application is configured to use `LongPolling`.